### DISCUSSION TITLE
Anura3D - compile failure

### DISCUSSION THREAD

**Discussion by DanBrox** [Anura3D/Anura3D_OpenSource#52] [Q&A] [answered]
Attempt to compile Anura3D code in Visual Studio results in an error stating the 'modemptyelements' mod file isn't found (attached).  Suggestions?  Notably, the Intel OneAPI Base Toolkit download had a problem with downloading the Intel Distribution for Python (attached) - don't know if this problem is related. 
![compileError](https://user-images.githubusercontent.com/46611376/204879242-52d0707c-04a6-4bfd-a0fa-6c596eacfc54.jpg)
![downloadError](https://user-images.githubusercontent.com/46611376/204879249-951703de-4e4c-47f7-aa3e-598d06b72c0b.jpg)
![PythonDistribution](https://user-images.githubusercontent.com/46611376/204879251-5e9cced7-f3d7-4bed-8fdd-c0e43a75fbb9.jpg)

**Comment by DanBrox** [comment, marked as answer]
This compile error was being caused by a misplaced line delimiter '&' in MPMEmptyElements.for:

![codeEdit](https://user-images.githubusercontent.com/46611376/208173961-f8c1116c-5334-463c-981d-5f3e7df3e5f8.jpg)

**Reply by murphy-john** [reply]
Yes, this was mentioned [here.](https://github.com/Anura3D/Anura3D_OpenSource/discussions/36#discussioncomment-3846613)

**Comment by DanBrox** [comment]
Thanks for the response.  Problem solved.
